wagtail_airtable/templatetags/wagtail_airtable_tags.py

```diff
@@ -0,0 +1,20 @@
+from django import template
+from django.conf import settings
+
+
+register = template.Library()
+
+@register.simple_tag
+def can_import_model(model_path) -> bool:
+    """
+    Check if a model can be imported based on it's model label.
```

Comment:
"it's" -> "its"

Is it model label or model path?

Reply:
Good catch! It's a model label. I'll update that function arg. Thanks!